crates/plugin_lodash/src/lib.rs

```diff
@@ -44,31 +54,45 @@ pub fn plugin_lodash(
     }
   });
 
-  let mappings = build_mappings(ids.iter().map(|s| s.as_str()), &config.cwd).unwrap();
-  let mut pkg_map = AHashMap::default();
+  let hash = config
+    .ids
+    .iter()
+    .fold(String::new(), |hash, curr| hash + curr);
+
+  {
+    let mappings_with_hash = CACHED.read();
+
+    if CACHED.read().2 != hash {
+      drop(mappings_with_hash);
```

Comment:
What's the use case of this? What do we need from reading this cache? It seems to me that only writing is really happening in this loop